fcli-core/fcli-aviator/src/main/java/com/fortify/cli/aviator/ssc/cli/cmd/AviatorSSCAuditCommand.java

```diff
@@ -63,24 +62,22 @@ public JsonNode getJsonNode(UnirestInstance unirest) {
                     SSCFileTransferHelper.ISSCAddDownloadTokenFunction.ROUTEPARAM_DOWNLOADTOKEN);
 
             progressWriter.writeProgress("Status: Processing FPR with Aviator");
-            File processedFile = AuditFPR.auditFPR(fprFile, sessionDescriptor.getAviatorToken(), sessionDescriptor.getAviatorUrl(), projectName, logger);
-            processedFile.deleteOnExit();
+            File processedFile = AuditFPR.auditFPR(fprFile, sessionDescriptor.getAviatorToken(), sessionDescriptor.getAviatorUrl(), appName, logger);
 
-            progressWriter.writeProgress("Status: Uploading FPR to SSC");
-            JsonNode uploadResponse = uploadFpr(unirest, processedFile, av);
-            JsonNode dataNode = uploadResponse.get("data");
-            String id = dataNode.has("id") ? dataNode.get("id").asText() : "";
-
-            return av.asObjectNode().put("artifactId", id);
-        } catch (AviatorSimpleException e) {
-            LOG.error("Aviator audit failed: {}", e.getMessage());
-            throw new FcliSimpleException(e.getMessage());
-        } catch (AviatorTechnicalException e) {
-            LOG.error("Technical error during Aviator audit: {}", e.getMessage(), e);
-            throw new FcliTechnicalException("Aviator audit failed due to a technical issue: " + e.getMessage(), e);
-        } catch (IOException e) {
-            LOG.error("I/O error during audit process: {}", e.getMessage(), e);
-            throw new FcliTechnicalException("Failed to process FPR file due to an I/O error.", e);
+            if (processedFile != null) {
+                processedFile.deleteOnExit();
+                progressWriter.writeProgress("Status: Uploading FPR to SSC");
+                JsonNode uploadResponse = uploadFpr(unirest, processedFile, av);
+                JsonNode dataNode = uploadResponse.get("data");
+                String id = dataNode.has("id") ? dataNode.get("id").asText() : "";
+                return av.asObjectNode().put("artifactId", id);
+            } else {
+                progressWriter.writeProgress("No issues to audit, skipping upload");
+                return av.asObjectNode().put("artifactId", "N/A").put("action","SKIPPED");
+            }
+        } catch (AviatorSimpleException | AviatorTechnicalException | IOException e) {
+            LOG.debug("Aviator audit failed: {}", e.getMessage(), e);
+            throw new FcliSimpleException("Failed to process FPR: " + e.getMessage());
```

Comment:
You shouldn't be translating technical/random exception types into `FcliSimpleException`, as that will suppress the stack traces. Also, for technical exceptions, you should include the original exception as the cause when wrapping an exception.